SU-SWS/stanford_profile
# D8CORE-704: Move image config.

# READY FOR REVIEW

# Summary
- Move the config to the right place.
- Add uuids.

# Review By (Date)
- October 25

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Run `blt drupal:install`
3. Validate the image styles are there and are working
4. Run tests

# Affected Projects or Products
- D8CORE-704

# Associated Issues and/or People
- D8CORE-704

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
